### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by PRAVEEN-mnl**
Hello, I did a small edit in this file from 'h5py>=2.10' to 'h5py==3.0.0'

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ scipy>=0.16.0
 matplotlib>=2.0.0
 numpy>=1.16.0,!=1.19.0
 pillow
-h5py>=2.10
+h5py==3.0.0
```
Comment (spxiwh) [inline requirements.txt:9]:
```suggestion
h5py>=3.0.0
```
We probably don't want to require *exactly* 3.0.0. You also want to make the same change in https://github.com/gwastro/pycbc/blob/master/setup.py#L45

**Comment by ahnitz** [comment]
@PRAVEEN-mnl Can you merge this with #3963  instead of having two separate PRs?

**Comment by tdent** [comment]
Superseded by #3967

**Comment by tdent** [comment]
FWIW, I think 'merging' two separate PRs together is already too complicated for someone's first example of dealing with a PR

**Pull request closed without merging**